### PR TITLE
Add code to automate parts of the dataset card

### PR DESCRIPTION
**Pull request by patrickvonplaten**
Most parts of the "Dataset Structure" section can be generated automatically. This PR adds some code to do so.

### CODE REVIEW

File: templates/README_guide.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ When relevant, please provide [BCP-47 codes](https://tools.ietf.org/html/bcp47),
 
 ## Dataset Structure
 
+IMPORTANT: This section can to some part be generated automatically as described in the end of the file [here](#create-dataset-structure).
```
Comment (lhoestq) [inline templates/README_guide.md:56]: If I understand correctly the tool is quite powerful when you have many configs and splits right ?
Then maybe for the general usage it's probably simpler to just write the paragraph manually.
Could we make it more explicit that this is for for a a more advanced usage and that it's optional ?
Let me know what you think

File: templates/README_guide.md [hunk 2]
```diff
@@ -184,3 +186,66 @@ Provide the [BibTex](http://www.bibtex.org/)-formatted reference for the dataset
...
+*e.g.*
+```
+./create_dataset_structure.py xglue mlqa,nc,ner,ntg,paws-x,pos,qadsm,qam,qg,wpr,xnli > dataset_section.txt
+```
+
+Note that in order to run the script, one should install `pytablewriter` and `json`.
+Content of `create_dataset_structure.py`:
+
+
+```python
```
Comment (lhoestq) [inline templates/README_guide.md:210]: awesome !

**Review by lhoestq — COMMENTED** [review]
Love it ! 
I left a comment about how to present this tool to the user in the guide

File: templates/README_guide.md (hunk 1 above)
Comment (patrickvonplaten) [reply templates/README_guide.md:56]: Definitely! Feel free to go into the PR and rewrite it to whatever you think better fits the library :-)

**Pull request closed without merging**